### PR TITLE
Features/browser support

### PR DESCRIPTION
**Pull request by JiriTrecak**
@artufimtcev Changes necessary for browser support

### CODE REVIEW

**Comment by JiriTrecak** [comment]
This will now work :)

File: src/tests/Tooling/PluginFigmaTokens.spec.ts [hunk 1]
```diff
@@ -68,31 +97,292 @@ test('test_tooling_design_tokens_load_and_merge_from_directory', async t => {
...
+  let tokenDefinition = await dataLoader.loadTokensFromPath(dataFilePath)
+  let configDefinition = dataLoader.loadConfigFromPath(mappingFilePath)
 
   // Run sync
-  await t.notThrowsAsync(syncTool.synchronizeTokensFromFile(dataFilePath, mappingFilePath))
+  await t.notThrowsAsync(syncTool.synchronizeTokensFromData(tokenDefinition, configDefinition.mapping, configDefinition.settings))
 })
+
+
+export class FigmaTokensDataLoader {
```
Comment (tchief) [inline src/tests/Tooling/PluginFigmaTokens.spec.ts:140]: @JiriTrecak Do you want to copy this into the `cli` or extract some code not dependent on fs into `DTJSONLoader` to reuse?

File: src/tools/design-tokens/utilities/SDKDTJSONLoader.ts [hunk 2]
```diff
@@ -69,6 +67,7 @@ export class DTJSONLoader {
   // MARK: - Loader
 
   /** Load token definitions from path */
+  /*
```
Comment (tchief) [inline src/tools/design-tokens/utilities/SDKDTJSONLoader.ts:70]: @JiriTrecak Should `DTJSONLoader` be removed or you consider having common code from `FigmaTokensDataLoader` not dependent on `fs` here?

File: src/tools/design-tokens/utilities/SDKDTMapLoader.ts [hunk 3]
```diff
@@ -75,6 +74,7 @@ export class DTMapLoader {
   // --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
   // MARK: - Mapping Conversion
 
+  /*
```
Comment (tchief) [inline src/tools/design-tokens/utilities/SDKDTMapLoader.ts:77]: @JiriTrecak Same here, `DTMapLoader` is empty class now

File: src/core/data/SDKDataBridge.ts [hunk 4]
```diff
@@ -71,50 +68,6 @@ export class DataBridge {
     this.requestHook = conf.requestHook
     this.debugRequestObserver = conf.debugRequestObserver ?? null
     this.debugResponseObserver = conf.debugResponseObserver ?? null
-
-    // Add performance interceptors once
```
Comment (tchief) [inline src/core/data/SDKDataBridge.ts:75]: @JiriTrecak Do we need to add this back later (log timings) or we don't really need this information?

**Review by tchief — COMMENTED** [review]
Main question is whether we try to
 - copy all of `FigmaTokensDataLoader` into cli, remove empty classes from SDK, add missing exports from SDK
 - or try to keep non-fs part in SDK, and copy into cli what is missed
 
I can update branch to any of this option, @JiriTrecak

**Comment by JiriTrecak** [comment]
@tchief let's do #1, that is what I wanted to do as well (which I why extracted the FigmaTokensDataLoader in the first place, it didn't exist before)

**Review by tchief — APPROVED** [review]

**Pull request merged**